src/hotspot/share/opto/node.hpp

```diff
@@ -1217,7 +1217,7 @@ class Node {
   // Print compact per-node info
   virtual void dump_compact_spec(outputStream *st) const { dump_spec(st); }
 
-  void verify_edges(Unique_Node_List &visited); // Verify bi-directional edges
+  static void verify_edges(Node* root, Unique_Node_List &visited, Node_List &nstack); // Verify bi-directional edges
```

Comment:
Maybe you can directly rename the method according to your comment: `verify_bidirectional_edges()`.